schemas/docfx.json

```diff
@@ -202,13 +202,7 @@
       "$ref": "#/definitions/OutputConfig"
     },
     "globalMetadata": {},
-    "siteBasePath": {
-      "type": "string"
-    },
-    "sourceBasePath": {
-      "type": "string"
-    },
-    "needGeneratePdfUrlTemplate": {
+    "showPdfUrlTemplate": {
```

Comment:
how about `output/pdf`? there is already an `output/json` property

Reply:
but pdf maybe not generated by docfx? do we have any plan for pdf and offline book?